scripts/functions/indexes/elasticsearch.py

```diff
@@ -7,10 +7,24 @@
 
 import requests
 import constant
+from simplejson import JSONDecodeError
+from simplejson import dumps
+
+# from requests.exceptions import JSONDecodeError
 #
 # @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
 #
 
+def prettyprint(result):
+    try:
+        pretty = dumps(result.json(), indent=2)
+        print(pretty)
+        return True
+    except JSONDecodeError as jsonE:
+        print('Response could not be JSON serialized')
+        print(result.text)
+        return True
```

Comment:
Why always True?

Reply:
removed, it's a print function, we don't need a return